OpenMEEG/libs/OpenMEEG/include/gain.h

```diff
@@ -213,13 +213,4 @@ namespace OpenMEEG {
         }
         ~GainStimInternalPot () {};
     };
-
-   class GainEITInternalPot : public Matrix {
-   public:
```

Comment:
as discussed with Theo, I should keep the GainEITInternalPot, and remove the GainStimInternalPot instead.
(this is not mentioned in the doc. only used once in openmeeg_sample_data)